### ISSUE TITLE
Add topics

### ISSUE THREAD

**Issue by tmeftah** [tmeftah/Aora#18]
- [x] #37
- [ ] #38
- [ ] #39
- [ ] #40

**Comment by tmeftah** [comment]
@NikhilRaikar17  you mixing the tasks. please focus on one part. The backend is not finished for adding topics!!!
i will set it backlog

**Comment by tmeftah** [comment]
@NikhilRaikar17  I added sub task. please solve each task separately. If you think more Sub-Task are needed add them.

**Comment by NikhilRaikar17** [comment]
@tmeftah As discussed, this ticket for now is paused from myside. I am concentrating on the backend for now. Once the docker topic is finished, I will update you.

**Comment by tmeftah** [comment]
@NikhilRaikar17 Yes but this ticket now have 4 sub tasks . 3 of them are  backend task and the lat one is UI. 
we will work both in this issue but each one of us will work on one or more sub tasks.
before we start let us talk.

**Comment by NikhilRaikar17** [comment]
CRUD Topics implementation started
